pipeline/transform.py

```diff
@@ -7,6 +7,8 @@
 DNB = ['Drum & Bass', 'Dnb', 'Drum N Bass']
```

Comment:
These are all good :). Have to remember that ideally would want to do this for a lot more genres but for now having this many is good